x-pack/plugins/observability/server/ui_settings.ts

```diff
@@ -245,11 +245,7 @@ export const uiSettings: Record<string, UiSettings> = {
     description: i18n.translate(
       'xpack.observability.enableInfrastructureProfilingIntegrationDescription',
       {
-        defaultMessage:
-          '{betaLabel} Enable Universal Profiling integration in the Infrastructure app.',
-        values: {
-          betaLabel: `<em>[${betaLabel}]</em>`,
-        },
+        defaultMessage: 'Enable Universal Profiling integration in the Infrastructure app.',
```

Comment:
Also not directly related, just removing the `beta` label as it's not applicable in this case.